python/sqlflow_submitter/xgboost/feature_column.py

```diff
@@ -0,0 +1,243 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+__all__ = [
+    'numeric_column',
```

Comment:
It seems that the API name is the same as TensorFlow feature column API. We can keep the name here at the current stage.
In the next step, we can rename the transform function in the COLUMN clause according to [the design doc](https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/data_transform.md) for TensorFlow/PyTorch/XGBoost.

Reply:
Agreed. We should rename them in the next step.